crates/oxc_minifier/tests/plugins/replace_global_defines.rs

```diff
@@ -42,3 +42,26 @@ fn replace_global_definitions_dot() {
         test("foo.process.NODE_ENV", "foo.process.NODE_ENV", config);
     }
 }
+
+#[test]
+fn replace_global_definitions_dot_with_postfix_wildcard() {
+    {
+        let config =
+            ReplaceGlobalDefinesConfig::new(&[("import.meta.env.*", "undefined")]).unwrap();
+        test("import.meta.env.result", "undefined", config.clone());
+        test("import.meta.env", "import.meta.env", config);
+    }
+}
+
+#[test]
+fn replace_global_definitions_dot_with_postfix_mixed() {
+    {
+        let config = ReplaceGlobalDefinesConfig::new(&[
+            ("import.meta.env.*", "undefined"),
+            ("import.meta.env", "env"),
+        ])
+        .unwrap();
+        test("import.meta.env.result", "undefined", config.clone());
+        test("import.meta.env", "env", config);
+    }
+}
```

Comment:
It seems like we are lacking tests that don't match the expression, can we add a few more to cover all the lines?